composer.json

```diff
@@ -9,7 +9,8 @@
 	},
 	"scripts": {
 		"post-autoload-dump": [
-			"php setup/cli.php build-artifacts --yes"
+			"php setup/cli.php build-artifacts --yes",
+			"libs/composer/vendor/bin/captainhook install -f"
```

Comment:
One concern: `CaptainHook` will most probably not be installed on productive environments, so `libs/composer/vendor/bin/captainhook` might not exist. Does this command result in any error if you execute `composer install --no-dev`?

Reply:
I found a good explanation about how to run this script only in dev mode here: https://stackoverflow.com/a/53022623/3655829

Reply:
Thanks for your hints.
 
You are right @mjansenDatabay, CaptainHook isn't present on live systems. I didn't think about that. So we have to find some workaround to avoid errors on composer runs with --no-dev flag. 
After reading the link posted by @PurHur i think we have a couple of potential solutions.

**Defining a condition in the command**
`"[ $COMPOSER_DEV_MODE -eq 1 ] && libs/composer/vendor/bin/captainhook install -f || echo 0"`
But that can cause problems on windows platforms, because $COMPOSER_DEV_MODE isn't available. Is that relevant?

**Install scriptsdev for a script-dev section in composer.json**
https://github.com/neronmoon/scriptsdev
Much cleaner way but one more dependency.

I think one of them could do the job for us. How should we proceed? Did i forgot something?

Reply:
I know there are some Windows users out there, e.g. @mbecker-databay  :-).

I am sure we will find a solution for them. I would like to discuss this change request in general on the next JourFixe. If approved, we can discuss possible solutions for Windows users.